app/src/main/java/de/hhn/aib/labsw/blackmirror/controller/API/Websockets/SendPackage.kt

```diff
@@ -0,0 +1,8 @@
+package de.hhn.aib.labsw.blackmirror.controller.API.Websockets
+
+import com.fasterxml.jackson.databind.JsonNode
+
+class SendPackage {
```

Comment:
please add an author and version

Reply:
replace this with a data class, see https://kotlinlang.org/docs/data-classes.html